docs/EN_US/ContainerizedHPCC/ContainerizedMods/CustomConfig.xml

```diff
@@ -992,4 +992,154 @@ eclagent:
       </sect3>
     </sect2>
   </sect1>
+
+  <sect1 id="LDAPSecurity_forConatiners">
+    <title>LDAP Security for Containers</title>
+
+    <para>Containerized LDAP support for HPCC Systems can be accomplished by
+    externalizing the LDAP Active Directory Security Manager administrator
+    account credentials using Kubernetes and Hashicorp Vault secrets. To use
+    these externalized credentials, as prerequisite should be familiar setting
+    up Kubernetes secrets and/or Hashicorp Vault secrets.</para>
+
+    <para>The LDAP Administrator account is the only HPCC user that must have
+    Active Directory administrator rights. This account should exist in the
+    configured "systemBasedn" branch of the Active Directory, typically set to
+    cn=Users.</para>
+
+    <sect2 id="Configuring_For_KubernetesSecrets">
+      <title>Configuring LDAP for Kubernetes</title>
+
+      <para>To configure secrets in Kubernetes, from the Kubernetes CLI,
+      create the LDAP "secret" similar to the following example. Note the
+      secret name, "admincredssecretname" used in this example. The LDAP
+      Administrator "username" and "password" key/values are required;
+      additional properties are allowed but ignored.</para>
+
+      <programlisting>kubectl create secret generic admincredssecretname --from-literal=username=hpcc_admin \
+  --from-literal=password=t0pS3cr3tP@ssw0rd
+
+kubectl get secret admincredssecretname
+</programlisting>
+
+      <para>This effectively sets the values for the Administrator's username
+      and password in Kubernetes.</para>
+
+      <para>For more information about Kubernetes see the appropriate
+      Kubernetes documentation for your implementation.</para>
+
+      <sect3 id="Deploying_TheKubernetesSecret">
+        <title>Deploying the Kubernetes Secret</title>
+
+        <para>Override the "secrets:" section in
+        HPCC-Platform/helm/hpcc/values.yaml, or your own customization
+        chart.</para>
+
+        <para>Create a unique key name used to reference the secret (this will
```

Comment:
I think this could be better explained, Maybe some steps? And specific relating of the value part of the K/V pair back to the created secret.